libs/paymentProcessor.js

```diff
@@ -779,7 +779,7 @@ function SetupForPool(logger, poolOptions, setupFinished){
                         }
 
                         // get the coin base generation tx
-                        const generationTx = tx.result.details.filter(tx => tx.address === poolOptions.address)[0]
+                        var generationTx = tx.result.details.filter(tx => tx.address === poolOptions.address)[0]
```

Comment:
Doh! I see it now. Good catch. I do have one small request:
```suggestion
                        let generationTx = tx.result.details.filter(tx => tx.address === poolOptions.address)[0]
```

Reply:
@shmocs Can you make this change? Thanks!

Reply:
sure